pkg/assembler/graphql/schema/package.graphql

```diff
@@ -181,8 +181,8 @@ extend type Query {
 }
 
 extend type Mutation {
-  "Ingests a new package and returns the corresponding package trie path. The returned ID can be empty string."
-  ingestPackage(pkg: PkgInputSpec!): ID!
-  "Bulk ingests packages and returns the list of corresponding package trie path. The returned array of IDs can be a an array of empty string."
-  ingestPackages(pkgs: [PkgInputSpec!]!): [ID!]!
+  "Ingests a new package and returns a corresponding package hierarchy containing only the IDs. The returned ID can be empty string."
+  ingestPackage(pkg: PkgInputSpec!): Package!
```

Comment:
based on the discussion in the slack, we should change this to return:
```
struct PackageIDs {
  packageTypeID: ID!
  packageNamespaceID: ID!
  packageNameID: ID!
  packageVersionID: ID!
}
```
so we are not capturing other fields unnecessarily during the ingestion. We only care about the IDs.

Reply:
Ah good catch, I had intended to go back and tidy this up.  We are only returning the IDs at present, albeit in the same hierarchy as Package.

Do you want this flattened or should we keep the hierarchy?  I was thinking the hierarchy could possibly be useful for merging responses at some future time.

Reply:
so we had that discussion here in Slack: https://openssf.slack.com/archives/C03U677QD46/p1697044972489989?thread_ts=1696877458.535239&cid=C03U677QD46. Based on that, we decided to go the flattened route as its simpler to implement.

Reply:
Strange, I didn't get notifications of that part of the discussion so had missed it.  I'll rework it